docs/_articles/en/soql/soql-builder.md

```diff
@@ -60,7 +67,27 @@ As you build your query, watch SOQL Builder display the query syntax while it si
 
 You can select objects and fields from the drop-down list, or type to narrow the list results. You can select an object or a field only once. If a value is already selected, it doesn't appear in the drop-down or search results.
 
-![Build your query](./images/soql-builder.gif)
+![Build your query](./images/soql-builder-build-a-query.gif)
+
+<!-- ekapner, 2/18/2021 - new gif with new name -->
+
+### Filter with the LIKE Operator <!-- ekapner, copy to JA version -->
+
+When filtering your results, you can narrow and target those results even further by using the LIKE operator to match partial text strings using wildcards. This query returns only last names that start with with `appl`.
```

Comment:
tiny suggestion, because this is a complex sentence.  "... **by using the LIKE operator with wildcards** to match partial text strings."  (bold is mine to show what I changed.).

Reply:
Appleby, Appleton, heh heh heh - went ahead and changed it. 

@jshackell-sfdc - great edits, thanks.